Microsoft.Toolkit.Uwp.UI.Controls/Menu/Menu.Extensions.cs

```diff
@@ -71,20 +71,20 @@ public static void SetInputGestureText(DependencyObject obj, string value)
         public static readonly DependencyProperty AllowTooltipProperty = DependencyProperty.RegisterAttached(AllowTooltipName, typeof(bool), typeof(Menu), new PropertyMetadata(false));
 
         /// <summary>
-        /// Gets InputGestureText attached property
+        /// Gets AllowTooltip attached property
         /// </summary>
-        /// <param name="obj">Target MenuFlyoutItem</param>
-        /// <returns>Input gesture text</returns>
+        /// <param name="obj">Target Menu</param>
+        /// <returns>AllowTooltip</returns>
         public static bool GetAllowTooltip(DependencyObject obj)
```

Comment:
It need to be Menu only, no need for menu item. will restrict it with the new changes